pages/franchise_page/[franchise].md

```diff
@@ -0,0 +1,200 @@
+    t."Photo URL" as logo,
+    t."Primary Color" as primary_color,
+    t."Secondary Color" as secondary_color,
+    case
+       when p."Franchise Staff Position" = 'NA' then 'Player'
+       else p."Franchise Staff Position"
+       end as franchise_position,
+    case when p."Franchise Staff Position" = 'Franchise Manager' then 1
+        when p."Franchise Staff Position" = 'General Manager' then 2
+        when p."Franchise Staff Position" = 'Assistant General Manager' then 3
+        when p."Franchise Staff Position" = 'Captain' then 4
+        when p."Franchise Staff Position" = 'Player' then 5
+        end as franchise_order
+ from players p
+    left join S17_stats s17
+        on p.member_id = s17.member_id
+    left join teams t
+        on p.franchise = t.Franchise
+    )
+  select distinct(name),
+  salary,
+  franchise,
+  logo,
+  league,
+  id_link,
+  franchise_position,
+  primary_color,
+  secondary_color
+  from playerstats
+  where franchise = '${params.franchise}'
+  and franchise_position = 'Franchise Manager' or
+  franchise = '${params.franchise}'
+  and franchise_position = 'General Manager' or
+  franchise = '${params.franchise}'
+  and franchise_position = 'Assistant General Manager'
+  order by franchise_order asc
+```
+
+> Franchise Staff Members
+<DataTable data={staff_members} rowshading=true headerColor='{team_info[0].primary_color}' headerFontColor=white>
+    <Column id=id_link contentType=link linkLabel=name align=center title=Player />
+    <Column id=salary align=center />
+    <Column id=league align=center />
+    <Column id=franchise_position align=center />
+</DataTable>
+
+```sql player_info
+select 
+name,
+'/player_page/' || p.member_id as id_link,
+salary,
+skill_group as league,
+franchise,
+SUBSTRING(slot, 7) as slot,
+doubles_uses,
+standard_uses,
+total_uses,
+current_scrim_points,
+case when current_scrim_points >= 30 then 'Yes'
+    else 'No'
+    end as Eligible
+from players p
+    inner join role_usages ru
+        on p.franchise = ru.team_name
+        and p.slot = ru.role
+        and upper(p.skill_group) = concat(ru.league, ' LEAGUE')
+where franchise = '${params.franchise}'
+and slot != 'NONE'
+and season_number = 17
+order by slot asc
+```
+
+```sql players
+select 
+name,
+'/player_page/' || p.member_id as id_link,
+salary,
+skill_group as league,
+franchise,
+SUBSTRING(slot, 7) as slot,
+doubles_uses,
+standard_uses,
+total_uses,
+current_scrim_points,
+case when current_scrim_points >= 30 then 'Yes'
+    else 'No'
+    end as Eligible
+from players p
+    inner join role_usages ru
+        on p.franchise = ru.team_name
+        and p.slot = ru.role
+        and upper(p.skill_group) = concat(ru.league, ' LEAGUE')
+where franchise = '${params.franchise}'
+and skill_group = '${inputs.League.value}'
+and slot != 'NONE'
+and season_number = 17
+order by slot asc
+```
+
+> League Selection
+<Dropdown data={player_info} name=League value=league />
+
```

Comment:
Add a Big Value here that shows the captain for the selected league.  I think that'll work nicer than the other ideas we didn't like very much.